cmake/OpenEXRSetup.cmake

```diff
@@ -246,6 +246,11 @@ endif()
 #######################################
 
 # Check to see if Imath is installed outside of the current build directory.
+set(IMATH_REPO "https://github.com/AcademySoftwareFoundation/Imath.git" CACHE STRING
+    "Repo for auto-build of Imath")
+set(IMATH_TAG "master" CACHE STRING
```

Comment:
Does it make sense to pin it to a specific version instead of using master?
What happens if I check out this state in one year - may be in the meanwhile something has changed which breaks the build, because the future Imath master does not work with this version - I would suggest pinning it to a specific version - same is done here: https://github.com/AcademySoftwareFoundation/openexr/blob/master/bazel/third_party/openexr.bzl 
The Bazel build could also use master for Imath - but this would make also testing in the CI random

Reply:
That's what the comment says right after that -- that it's "master" but needs to be changed by the release, so point to a specific tag.

Reply:
Line 253:

```
#TODO: ^^ Release should not clone from master, this is a place holder
```

Reply:
Sorry - I should read everything ;)